src/index.ts

```diff
@@ -506,7 +506,11 @@ function validate(...args: string[]): void {
     if (arg.split('-').length - 1 > 1) {
```

Comment:
Is two negative signs possible given scientific notation like `-12e-2`? I doubt it, but if we ever see this error message crop up, we know possibly why.